plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions/VariationSelectorItemTemplate.php

```diff
@@ -50,13 +50,30 @@ protected function render( $attributes, $content, $block ): string {
 	 *
 	 * @param string   $product_attribute_name Product Attribute Name.
 	 * @param array    $product_attribute_terms Product Attribute Terms.
-	 * @param array    $attributes Block attributes.
 	 * @param WP_Block $block The Block.
 	 * @return string Row HTML
 	 */
-	private function get_product_row( $product_attribute_name, $product_attribute_terms, $attributes, $block ): string {
-		$attribute_name  = $product_attribute_name;
-		$attribute_terms = $this->get_terms( $product_attribute_name, $product_attribute_terms );
+	private function get_product_row( $product_attribute_name, $product_attribute_terms, $block ): string {
+		global $product;
+
+		$attribute_name     = $product_attribute_name;
+		$attribute_terms    = $this->get_terms( $attribute_name, $product_attribute_terms );
```

Comment:
Not related to this PR, but I guess that the PHPDoc of `get_terms` function is not correct (see return type - it is `srtring`): https://github.com/woocommerce/woocommerce/blob/b10c01e48fd5ecece0dec0a6a7e5ad31fb86deb7/plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions/VariationSelectorItemTemplate.php#L96-L103

Reply:
Ups, good catch. I totally missed it! Should be fixed in: c417e5821171ce010981f8345d72939d22889c8a.